src/modules/cmdpal/extensionsdk/Microsoft.CommandPalette.Extensions/Microsoft.CommandPalette.Extensions.idl

```diff
@@ -273,10 +273,19 @@ namespace Microsoft.CommandPalette.Extensions
         String Section { get; };
         String TextToSuggest { get; };
     }
-    
+
+    enum GridTileSize
+    {
+        Small,
+        Medium,
+        Large
+    };
+
     [contract(Microsoft.CommandPalette.Extensions.ExtensionsContract, 1)]
-    interface IGridProperties  {
-        Windows.Foundation.Size TileSize { get; };
+    interface IGridProperties {
+        GridTileSize Size { get; };
```

Comment:
huh, didn't like the "developers can pick an arbitrary size in DIPs for their items" I see? was there something that was particularly fucky about that? Or just a convenience thing (like, folks would likely pick different sizes so the grids would all look different)